views/participation/activities.erb

```diff
@@ -0,0 +1,46 @@
+<div class="hero">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-6">
+        <h1>Lepšie digitálne služby štátu <strong>nie sú</strong> sci-fi</h1>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div id="activities" class="section">
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12">
+        <h2 class="text-center">Zoznam aktivít Slovensko.Digital</h2>
+      </div>
+    </div>
+    <div id="filter" class="row mb-5">
+      <div class="col-xs-12 text-center">
+        <div class="btn-group hidden-xs hidden-sm" role="group">
```

Comment:
markup a obsah je pri `btn-group hidden-xs hidden-sm` / `btn-group-vertical visible-xs visible-sm` uplne rovnaky, to by mohlo vyriesit scss (podmieneny `@extend`) alebo mozno takto:

```
<div class="btn-group-vertical visible-xs visible-sm" role="group">
  <div class="btn-group hidden-xs hidden-sm" role="group">
    ...
  </div>
</div>
```